app/uk/bl/Const.java

```diff
@@ -82,7 +82,8 @@ private Const() {}
 	public static final String TARGET_DEF         = "TARGETDEF";
 	public static final String OFFSET             = "offset";
 	public static final String LIMIT              = "limit";
-	public static final String LIST_DELIMITER     = "; ";//", ";
+	public static final String LIST_DELIMITER     = ";";//", ";
+	public static final String SEED_URLS_DELIMITER    = ", ";
```

Comment:
I'm don't believe a space should be enforced here. I think you need to split on ',' then trim leading/trailing whitespace.

Reply:
Are we forcing users to use `comma-space` as a separator? I don't think we should do that.